Assets/MRTK/Core/Providers/InputSimulation/MixedRealityInputSimulationProfile.cs

```diff
@@ -183,14 +183,16 @@ public class MixedRealityInputSimulationProfile : BaseMixedRealityProfile
         /// </summary>
         public string LookVertical => lookVertical;
 
-        [Header("Eye Simulation")]
+        [Header("Eye Gaze Simulation")]
         [SerializeField]
-        [Tooltip("Enable eye simulation")]
-        private bool simulateEyePosition = false;
+        [Tooltip("Enable eye gaze simulation")]
+        [FormerlySerializedAs("eyeGazeSimulationMode")]
```

Comment:
I can't find eyeGazeSimulationMode as a thing that existed in the past (i.e. is this doing anything?)

Reply:
Ah, probably should just be simulateEyePosition

Reply:
Just trying to simplify the eye simulation things from 2 boolean variables to a single enum

Reply:
I actually had a concern about getting rid of this old one (i.e. simulateEyePosition property) but I guess because bool == true == 1 and bool == false == 0 actually map pretty well, I think this ends up being just fine.

You'll probably wanna validate this, but if you instead do FormerlySerializedAs("simulateEyePosition") it MIGHT be okay.

The only other thing to mention is that you'd probably want to keep the old public bool (mark it as Obsolete) and then make it just map to defaultEyeGazeSimulationMode == CameraForwardAxis i..e SimulateEyePosition => defaultEyeGazeSimulationMode  == CameraForwardAxis or SimulateEyePosition => defaultEyeGazeSimulationMode  != Disabled)

Reply:
Yeah, I'm kinda confused what the `FormerlySerializedAs` name is referring to. I don't see anything that was previously called `eyeGazeSimulationMode`. Should this be `[FormerlySerializedAs("simulateEyePosition ")]`?